internal/testfixtures/db.go

```diff
@@ -28,6 +28,8 @@ var DefaultDBConfig = DBConfig{
 func init() {
 	if pw := os.Getenv("DB_PASSWORD"); pw != "" {
 		DefaultDBConfig.Password = pw
+	} else {
```

Comment:
We can change the password to just always be dev and update the docker container on here?

Reply:
Yes